nulib/dc-nextjs
# Update Viewer configuration on /embedded-viewer route to display the meta…

## What does this do?
Shows the metadata side panel in a Viewer on a `/embedded-viewer` item route.

![image](https://github.com/nulib/dc-nextjs/assets/3020266/e82549c1-287b-4de5-aa95-b4d96e4a80f2)


## How to test
1. In DC go to any Work and click "Download and Share", and then copy the `<iframe src` attribute value as seen below:

![image](https://github.com/nulib/dc-nextjs/assets/3020266/6b16ff39-aaa6-453c-92ea-fe9e569f7319)

2. Copy this URL into a browser window.
3. Notice the metadata panel in the Viewer is open
4. View a regular Work in DC, and notice the metadata panel in the Viewer is not open.
